lib/smart_answer_flows/marriage-abroad/outcomes/outcome_same_sex_marriage_and_civil_partnership.govspeak.erb

```diff
@@ -139,8 +139,8 @@
              locals: { calculator: calculator } %>
 
   <% if calculator.ceremony_country == 'vietnam' %>
-    <%= render partial: 'consular_fees_in_cash_visa_master.govspeak.erb',
-               locals: { calculator: calculator } %>
+    <%= render partial: 'link_to_consular_fees_pay_in_sterling_or_local_currency.govspeak.erb',
+                locals: { calculator: calculator } %>
```

Comment:
It's incredibly minor but it'd be great to avoid these sort of unnecessary whitespace changes.